blocks/container/Affix.yaml

```diff
@@ -0,0 +1,64 @@
+id: Affix
+type: PageSiderMenu
+public: true
+properties:
+  title: Affix
+actions:
+  onInit:
+    - id: init
+      type: setState()
+      params:
+        _ref: blocks/html_blocks.yaml
+blocks:
+  _ref:
+    path: blocks/flex_template.yaml
+    vars:
+      block_type: Affix
+      block_name: Affix
+      page_description: |
+        Wrap an Affix block around other blocks to make it stick to the viewport.
+
+      style_transform:
+        _parse:
+          _state: flex_style_input
+      style_form:
+        - id: flex_style_input
+          type: TextArea
+          style:
+            _global: settings_input_style
+          defaultValue: |
+            {
+              "Content": {
+                "--display": "flex"
+              } 
+            }
+          properties:
+            title: block
+            rows: 3
+            size: small
+            Label:
+              extra: CSS Object style properties. Should be in JSON format.
+      flex_properties_form:
+        - id: block.properties.offsetBottom
+          type: NumberInput
+          style:
+            _global: settings_input_style
+          properties:
+            title: offsetBottom
+            size: small
+            Label:
+              extra: Offset from the bottom of the viewport (in pixels).
+        - id: block.properties.offsetTop
+          type: NumberInput
+          style:
+            _global: settings_input_style
+          properties:
+            title: offsetBottom
+            size: small
+            Label:
+              extra: Offset from the top of the viewport (in pixels).
+        
+      actions_description: | 
+        #### onChange
+        Called when Affix state is changed.
```

Comment:
We could add more detail about what affix state is. Easy to confuse with page/blocks state